app/models/concerns/hyrax/ability.rb

```diff
@@ -188,5 +193,15 @@ def user_is_depositor?(document_id)
       def curation_concerns_models
         [::FileSet, ::Collection] + Hyrax.config.curation_concerns
       end
+
+      def can_review_submissions?
+        # Are there any workflows where this user has the "approving" responsibility
+        approving_role = Sipity::Role.find_by(name: 'approving'.freeze)
+        return false unless approving_role
+        Hyrax::Workflow::PermissionQuery.scope_processing_agents_for(user: current_user).any? do |agent|
```

Comment:
As I've been thinking about this it sounds like we need the following (so as to avoid magic role names that may or may not exist):

This method should return true if the given user has one or more roles in a workflow that are at the workflow level (and not the work specific level).

However, consider if someone is granted permission to review a specific work, they would perhaps want a dashboard for review. I personally believe rethinking this UI as "My Tasks" is a better mechanism. Then any authenticated user has the page. In many cases it is empty. We could also query "How many tasks do I have to work on" and put a numeric badge by "My Tasks".